src/components/TopBar/TopBar.vue

```diff
@@ -282,6 +297,37 @@ export default {
 		},
 	},
 
+	watch: {
+		unreadMessagesCounter(newValue, oldValue) {
+			if (!this.isInCall || !this.showOpenSidebarButton) {
+				return
+			}
+
+			// new messages arrived
+			if (newValue > 0 && oldValue === 0) {
+				if (this.hasUnreadMentions) {
+					showMessage(t('spreed', 'You have been mentioned in the chat.'))
+				} else {
+					showMessage(t('spreed', 'You have new unread messages in the chat.'))
+				}
+			}
+		},
+
+		hasUnreadMentions(newValue, oldValue) {
+			if (!this.isInCall || !this.showOpenSidebarButton) {
+				return
+			}
+
+			// in some cases the user already had unread messages and saw the other notification
+			// but then got mentionned
+			// the limit of "> 1" is here to avoid seeing two notifications, one for the first non-mention
+			// unread message and the second for the mention
```

Comment:
This would be okay from my POV (second showMessage when you are now mentioned instead of unread chat)

Reply:
done